tests/common/test_cgroupapi.py

```diff
@@ -27,6 +27,8 @@
 from azurelinuxagent.common.future import ustr
```

Comment:
At some point I think I clicked on "format file" so there's a lot of whitespace updates, I'll point out relevant updates to the tests.